esque/validation/validators.py

```diff
@@ -0,0 +1,71 @@
+import re
+from yamale.validators import Boolean, DefaultValidators, Integer, Number, Validator
+from yamale.util import isstr
+
+
+def all_validators() -> dict:
+    validators = DefaultValidators.copy()
+    validators[StringBool.tag] = StringBool
+    validators[StringInt.tag] = StringInt
+    validators[StringFloat.tag] = StringFloat
+    validators[StringDictionary.tag] = StringDictionary
+
+    return validators
+
+
+def catch_value_errors(validate):
+    """method decorator to catch ValueErrors for casts and return an error"""
+
+    def wrapper(self, value):
```

Comment:
```suggestion
    @wraps(validate)
    def wrapper(self, value):
```
`wraps` is from the `functools` module.